src/søk/søkefelt/urlUtils.ts

```diff
@@ -59,3 +66,18 @@ export const byggUrlMedParam = (param: QueryParam, value: QueryParamValue) => {
 
     return url;
 };
+
+export const oppdaterUrlMedParam = ({
```

Comment:
Alle stedene der vi brukte `byggUrlMedParam` måtte vi også replace URL. Så laget en egen funksjon for dette. Også er det vel en tommelfingerregel som sier at med flere enn _x_ argumenter bør du legge disse i et objekt og navngi dem